### PR TITLE
Adds Overwatch API support

### PR DESCRIPTION
**Pull request by oxhacks**
Removes old Overwatch API implementation, adds a new one and puts it in the template.

Future MR will add the toggle for "only show the stats for this game when it is being played."

### CODE REVIEW

File: games/pubg.py [hunk 1]
```diff
@@ -85,4 +85,8 @@ def stats(name):
     :returns: the simplified stats for a given account name
 
     """
-    return get_stats_simple(name)
+    try:
+        response = get_stats_simple(name)
+    except json.decoder.JSONDecodeError as exc:
```
Comment (arusahni) [inline games/pubg.py:91]: Unused `exc`

File: templates/index.html [hunk 2]
```diff
@@ -72,12 +72,23 @@ <h5 class="light-key"><b>Playing:</b> <span class="trim">{{ s.playing }}</span><
           </div>
           {% if s.pubg %}
           <div class="flexed">
+            <div class="light-text"><em>PUBG </em></div>
+            //
```
Comment (arusahni) [inline templates/index.html]: Is this intentional?

File: requirements.txt [hunk 3]
```diff
@@ -12,4 +12,4 @@ visitor==0.1.3
 Werkzeug==0.11.11
 humanize==0.5.1
 pypubg==0.1.5
-overwatch-api==0.5
+python-overwatch=0.9.1
```
Comment (arusahni) [inline requirements.txt]: Needs to be a `==`.

File: templates/index.html (hunk 2 above)
Comment (oxhacks) [reply templates/index.html]: Not at all. Fixed.

**Comment by arusahni** [comment]
Some questions:

* The new Overwatch library introduces a BS4 dependency. Is this something we want on the response thread?
* Something is causing Werkzeug to log `INFO` messages in the dev server output. Can we tamp those back down? Current: 
image old: 
image

File: requirements.txt (hunk 3 above)
Comment (oxhacks) [reply requirements.txt]: Fixed

File: games/pubg.py (hunk 1 above)
Comment (oxhacks) [reply games/pubg.py:91]: Fixed

**Comment by oxhacks** [comment]
@arusahni I've resolved the nits you pointed out, all of which were valid. Having BS4 in the loop is not ideal for several reasons, but the library worked well enough otherwise that I was willing to trade off. 

I also have forked that library to remove the overwrought Daiquiri logger that was causing all of that INFO dump.

**Comment by arusahni** [comment]
Awesome. Thanks for going the extra mile w/the fork!

**Pull request merged**